riotctrl/__init__.py

```diff
@@ -8,4 +8,4 @@
 and maybe also over network.
 """
 
-__version__ = '0.4.1'
+__version__ = '0.5.0'
```

Comment:
seems unrelated

Reply:
See OP...

> In addition, I added a version bump. It's about time for a new version: last one was in March, riotctrl.flash() was introduced since then and the README heavily amended (which will show up on pypi as the package description).